oak/server/logging_node.cc

```diff
@@ -18,9 +18,12 @@
 
 #include "absl/memory/memory.h"
 #include "asylo/util/logging.h"
+#include "oak/proto/logger.pb.h"
 
 namespace oak {
 
+const std::string LoggingNode::log_levels[5] = {"ERROR", "WARN", "INFO", "DEBUG", "TRACE"};
```

Comment:
Is there any way of getting this string representation out of an actual enum value?

Reply:
Yes. Fixed using Level_Name function.